src/rpc/rpcevo.cpp

```diff
@@ -942,7 +941,6 @@ static UniValue BuildDMNListEntry(CWallet* pwallet, const CDeterministicMNCPtr&
     if (pwallet) {
         UniValue walletObj(UniValue::VOBJ);
         walletObj.pushKV("hasOwnerKey", hasOwnerKey);
-        walletObj.pushKV("hasOperatorKey", hasOperatorKey);
```

Comment:
this was kept for backwards (and maybe future) compatibility iirc, I would suggest to still keep it but simply as `walletObj.pushKV("hasOperatorKey", false);` for now maybe

Reply:
I'm not really sure this is needed, but sure, done